models/public/vgg19-cf2/model.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  This is a Caffe2\* version of `vgg19` model, designed to perform image classification.
+  This model was converted from Caffe\* to Caffe2\* fromat. 
+  For details see repository <https://github.com/caffe2/models/tree/master/vgg19>,
+  paper <https://arxiv.org/pdf/1409.1556.pdf>.
+task_type: classification
+files:
+  - name: predict_net.pb
+    size: 2862
+    sha256: ebb8608fe80ee8bce096a60ecf3e6e8442eb118aaa0fa77d5d58b5dcff7dfb5f
+    source: https://s3.amazonaws.com/download.caffe2.ai/models/vgg19/predict_net.pb
+  - name: init_net.pb
+    size: 718338501
+    sha256: 492dbbbc7dd23cb052c66964714759f88370f3fa8542aa32556e93abc7beb69f
+    source: https://s3.amazonaws.com/download.caffe2.ai/models/vgg19/init_net.pb
+framework: caffe2
+caffe2_to_onnx:
```

Comment:
Wouldn't it be better to have unified field e.g. `convert_to_onnx` in the config and understand in the script what to convert based on the framework?
with the increase in the number of frameworks that require conversion to ONNX, we get an increase in script parameters that are unlikely to ever be used together (e.g. `pytorch_to_onnx_args`, `caffe2_to_onnx` args, `cntk_to_onnx_args` and so on)

Reply:
+0, but it ought to be named something like `onnx_conversion_args`, for consistency with `model_optimizer_args`.